install_files/ansible-base/roles/tails-config/tasks/migrate_existing_config.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Between 0.3.12 and 0.4, the "Document Interface" was renamed to
+# "Journalist Interface". Since multiple scripts expect to readfrom this file,
+# we'll rename the old file to handle the transition gracefully.
+- name: Migrate Document Interface ATHS file (upgrade only).
+  command: >-
+    mv
+    "{{ tails_config_ansible_base }}/app-document-aths"
+    "{{ tails_config_ansible_base }}/app-journalist-aths"
+  args:
+    # Only run this command if the deprecated file is present.
+    removes: "{{ tails_config_ansible_base }}/app-document-aths"
+
+# Between 0.3.12 and 0.4, the `prod-specific.yml` vars file was deprecated
+# in favor of `group_vars/all/site-specific`. The latter is not version
+# controlled, and so will not exist by default.
+- name: Migrate site-specific configuration to new location.
+  command: >-
+    cp
```

Comment:
feel free to rebase back and banish that `919c3a1` commit, i have no issues with that specific file manipulation since the `app-document-aths` file is still ignored by git where as the `prod-specific.yml` file will continue be a thorn in the future (ie - the whole stash and pop dance).